pkg/planner/core/casetest/partition/testdata/integration_partition_suite_out.json

```diff
@@ -91,24 +91,14 @@
       {
         "SQL": "explain format = 'brief' select * from tlist where a not in (0, 1, 2, 3, 4, 5, 6, 7, 8)",
         "DynamicPlan": [
-          "TableReader 3583.33 root partition:all data:Selection",
+          "TableReader 3583.33 root partition:p3 data:Selection",
```

Comment:
Does the range optimizer check the list and returns two ranges (<0 and >8), which results in the new pruning result?

Reply:
Yes.